src/Core/Checkout/Document/Service/DocumentConfigLoader.php

```diff
@@ -60,6 +64,13 @@ public function load(string $documentType, string $salesChannelId, Context $cont
 
         $config = DocumentConfigurationFactory::createConfiguration([], $globalConfig, $salesChannelConfig);
 
+        if (Uuid::isValid($config->getCompanyCountryId())) {
+            /** @var ?CountryEntity $country */
```

Comment:
add the generic annotation to the entity repos and you do not need this var annotation